src/main.rs

```diff
@@ -80,10 +85,16 @@ fn main() -> anyhow::Result<()> {
 
         let path = file.path();
 
+        #[cfg(target_os = "linux")]
```

Comment:
Could you change this to `#[cfg(not(target_of = "windows"))]` so this also works on mac?